examples/user_interaction/client/src/App.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      gRPC://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import './App.css';
+
+class App extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { pi: { time: 0, value: "unknown" }, prime_numbers: { time: 0, value: [] } };
+    this.handleClick = this.handleClick.bind(this);
+    this.http = require('http');
+  }
+
+  handleClick() {
+    const host = 'localhost';
+    const port = 8088;
+    this.http.get({
+      host,
+      port,
+      path: '/sleep'
+    }, (response) => {
+      let data = [];
+      response.on('data', chunk => data.push(chunk));
+      response.on('end', () => {
```

Comment:
What would you think about making this a separate function to reduce the nesting of the code?